tests/components/recorder/test_purge.py

```diff
@@ -1552,9 +1552,7 @@ async def test_purge_many_old_events(
 
     with session_scope(hass=hass) as session:
         events = session.query(Events).filter(Events.event_type.like("EVENT_TEST%"))
-        event_datas = session.query(EventData)
```

Comment:
This test should have never been looking at event data as its testing old events without it